packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_world_map_card.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+import 'package:openfoodfacts/model/KnowledgePanelElement.dart';
+import 'package:smooth_ui_library/util/ui_helpers.dart';
+
+class KnowledgePanelWorldMapCard extends StatelessWidget {
+  const KnowledgePanelWorldMapCard(this.mapElement);
+
+  final KnowledgePanelWorldMapElement mapElement;
+
+  @override
+  Widget build(BuildContext context) {
+    if (mapElement.pointers.isEmpty || mapElement.pointers.first.geo == null) {
+      return EMPTY_WIDGET;
+    }
+    return Padding(
+      padding: const EdgeInsets.only(bottom: MEDIUM_SPACE),
+      child: SizedBox(
+        height: 200,
+        child: FlutterMap(
+          options: MapOptions(
+            // The first pointer is used as the center of the map.
+            center: LatLng(
+              mapElement.pointers.first.geo!.lat,
+              mapElement.pointers.first.geo!.lng,
+            ),
+            zoom: 13.0,
+          ),
+          layers: <LayerOptions>[
+            TileLayerOptions(
+              urlTemplate: 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png',
+              subdomains: <String>['a', 'b', 'c'],
+            ),
```

Comment:
I assume there should be a copyright notice visible on the map. I used to work on osmdroid ;)

Reply:
tbh I don't know how to add the copyright on the map itself, do you know if there's a way ? If not, is there another way of attributing, perhaps in text under the map.

Reply:
Let me take care of the maps after this PR.